app/res/values/styles.xml

```diff
@@ -328,7 +328,7 @@
     <style name="CustomButtonStyleOutline" parent="Widget.MaterialComponents.Button.OutlinedButton">
         <item name="shapeAppearanceOverlay">@style/RoundedButtonStyle</item>
     </style>
-    <style name="RoundedButtonStyle" parent="">
+    <style name="RoundedButtonStyle" parent="Widget.MaterialComponents.Button.OutlinedButton">

```

Comment:
Are you sure that `Widget.MaterialComponents.Button.OutlinedButton` is the right style to be applied ? And Have you tested impact of changing the parent Style here ?

Reply:
@shubham1g5 this is used to set the `shapeAppearanceOverlay` of the `CustomButtonStyleOutline` style so it should be fine. And yes, I tested it and it seemed fine, applied to the _CANCEL_ button below:
![image](https://github.com/user-attachments/assets/b33cf853-67cf-4a45-a02a-954ac8042b20)

Reply:
ahh can we rename this style to suggest that it's only applied for alert dialogs. `RoundedButtonStyle` seems too generic if it's only for dialogs.

Reply:
curious why do we need to apply it to the clear button  ?

Reply:
I have no context about that decision, it's the same style as before. I believe it's considered a negative action